### ISSUE TITLE
Actionable items from `@snoop_llvm` in the tutorial

### ISSUE THREAD

**Issue by Sbozzolo** [timholy/SnoopCompile.jl#399]
I am excited to see that there's a [Tutorial on `@snoop_llvm`](https://timholy.github.io/SnoopCompile.jl/stable/tutorials/snoop_llvm/), but I am finding the tutorial not very informative. In particular, the tutorial does not explain the relevance of the the data saved in the files and does not tell me how I can interpret the data and take action to reduce LLVM time.

**Comment by timholy** [comment]
You're right that the docs are just a stub. I've not really used this myself, so I didn't spend the time to explore & document it thoroughly. CC @NHDaly.
